libnavui-maps/api/current.txt

```diff
@@ -1184,17 +1184,6 @@ package com.mapbox.navigation.ui.maps.route.line.model {
     method public com.mapbox.maps.extension.style.expressions.generated.Expression generateExpression();
   }
 
-  public final class RouteLineGranularDistances {
```

Comment:
I'm intentionally removing this data class from the public API. It should've ever been exposed, and it has no practical usage, it's only consumed by internal Nav SDK APIs.